src/Pages/login.tsx

```diff
@@ -1,23 +1,44 @@
-import { ChangeEvent, useState } from "react";
-import { useNavigate } from "react-router-dom";
+import axios from "axios";
+import { ChangeEvent, FormEvent, useState } from "react";
+import { useLocation, useNavigate } from "react-router-dom";
 
 const Login = () => {
-    const [username, setUserName] = useState<string>('');
+    const [email, setEmail] = useState<string>('');
```

Comment:
@ShiranthaKellum, we don't need these states, we can capture username and password values from the form submit event, please refer below links,

- https://stackoverflow.com/questions/3547035/getting-html-form-values
- https://www.designcise.com/web/tutorial/how-to-access-form-control-elements-in-the-onsubmit-event-handler-in-react